Source/SwiftLintFramework/Extensions/SwiftLintFile+Regex.swift

```diff
@@ -61,8 +61,10 @@ extension SwiftLintFile {
         let contents = self.contents.bridge()
         let range = range ?? NSRange(location: 0, length: contents.length)
         let pattern = "swiftlint:(enable|disable)(:previous|:this|:next)?\\ [^\\n]+"
-        return match(pattern: pattern, with: [.comment], range: range).compactMap { range in
-            return Command(string: contents, range: range)
+        return match(pattern: pattern, range: range).filter { match in
+            return Set(match.1).isSubset(of: [.comment, .commentURL])
```

Comment:
maybe we should use `SyntaxKind.commentKinds` just to be safe?

Reply:
`SyntaxKind.commentKinds` contains
* `.comment`,
* `.commentUrl`,
* `.docComment`,
* `.docCommentField`, and
*  `.commentMark`.

Changing this as proposed would allow adding SwiftLint commands to doc comments as well as `MARK:` comments, which is currently not supported. I'm not sure whether such a change should be part of this PR.